JustSaying/AwsTools/MessageHandling/SnsTopicBase.cs

```diff
@@ -71,13 +74,15 @@ public void Publish(Message message)
         }
 #endif
 
-        public async Task PublishAsync(Message message)
+        public async Task PublishAsync(Message message) => await PublishAsync(message, CancellationToken.None);
```

Comment:
I'm going crazy, I swear you already changed this. Please return the task, or configureawait(false)